include/vrapp/VRVolumeApp.h

```diff
@@ -152,7 +174,20 @@ class VRVolumeApp
 
   std::vector< Volume* >& get_volume(int volume);
 
-  
+  ArcBallCamera& get_trackball_camera();
+
+  void set_animation_speed(float time);
```

Comment:
I think of speed as a rate, but it looks like the variable name is a time (or length).  How does the length and frame rate of the animation get set? Are users able to set a number of seconds (or frames) between POIs?

Reply:
This variable manage the speed of the volume animation ( transition between multiple datasets over a period of time), not the camera and slices animations. Although, it's possible to modify the speed of the volume animation, it is using predefined values by the app. The user cant set specific frames/second.

Reply:
I'm still not sure I'm following what the `time` value represents here.  Is it a number of frames? a number of seconds? other?

Reply:
I renamed it as `set_volume_animation_scale_factor`. It just scales the step size of the animation. I still need out the animation  by fps.